settings.gradle

```diff
@@ -1,6 +1,6 @@
 rootProject.name = 'fw-monitoring-examples'
 include 'ktor-monitoring'
 include 'spring-boot-monitoring'
+include 'spring-boot-monitoring-kt'
```

Comment:
is possible to follow proposed naming "framework"-"language"-"purpose", example spring-boot-kt-monitoring ?